src/app/static/src/app/components/plots/Choropleth.vue

```diff
@@ -126,6 +150,34 @@
                 }
 
                 return data;
+            },
+            getFeatureFromAreaId(areaId: string){
+                return (this.features as any[]).filter(f => f.properties.area_id == areaId)[0];
+            },
+            updateBounds(){
+                Vue.nextTick().then(() => {
+                    const geoJsonArray = this.$refs.selectedRegionsGeoJson;
```

Comment:
I think this whole thing can be replaced with:
```
                Vue.nextTick().then(() => {
                    const map = this.$refs.map;                  
                    map.fitBounds(this.selectedRegionFeatures.map((f: Feature) => new GeoJSON(f).getBounds()));
                });
```
and the ghost features removed from the map

Reply:
Oh I guess you don't need the nextTick bit at all either!

Reply:
:facepalm: Thank you! I tried and failed to get something like that working, think I was looking in the wrong place for the right GeoJSON class..